Modules/Segmentation/Voronoi/include/itkVoronoiDiagram2DGenerator.hxx

```diff
@@ -1073,19 +1073,12 @@ VoronoiDiagram2DGenerator<TCoordRepType>::GenerateVDFortune()
   m_BottomSite = &(m_SeedSites[0]);
   FortuneSite * currentSite = &(m_SeedSites[1]);
 
-  PointType         currentCircle{};
   FortuneHalfEdge * leftHalfEdge;
   FortuneHalfEdge * rightHalfEdge;
-  FortuneHalfEdge * left2HalfEdge;
-  FortuneHalfEdge * right2HalfEdge;
   FortuneHalfEdge * newHE;
   FortuneSite *     findSite;
-  FortuneSite *     topSite;
-  FortuneSite *     saveSite;
   FortuneEdge *     newEdge;
   FortuneSite *     meetSite;
-  FortuneSite *     newVert;
-  bool              saveBool;
```

Comment:
First of all, I like style commits that remove unused code 👍 Do you possibly have an explanation, why so much of the code of VoronoiDiagram2DGenerator seems unused or useless? Is it unfinished work?